packages/dds/tree/src/feature-libraries/chunked-forest/codec/compressedEncode.ts

```diff
@@ -66,7 +66,7 @@ export type Shape = ShapeGeneric<EncodedChunkShape>;
  */
 export interface KeyedFieldEncoder {
 	readonly key: FieldKey;
-	readonly shape: FieldEncoder;
+	readonly encoder: FieldEncoder;
```

Comment:
This change looks good.

I gave the doc comments for FieldEncoder to make sure we weren't losing any useful implications about it encoding a specific "shape" from this rename, and it links back to NodeEncoder which says `An encoder for a specific shape of node.` so I think those semantics are preserved while using this much clearer name,